core/services/synchronization/telemetry_ingress_batch_client.go

```diff
@@ -144,6 +169,10 @@ func (tc *telemetryIngressBatchClient) getCSAPrivateKey() (privkey []byte, err e
 // the ingress server. If the worker telemetry buffer is full, messages are dropped
 // and a warning is logged.
 func (tc *telemetryIngressBatchClient) Send(payload TelemPayload) {
+	if tc.useUniConn && !tc.connected.Load() {
+		tc.lggr.Errorw("not connected to telemetry endpoint", "endpoint", tc.url.String())
```

Comment:
what do you think about switching this to a warning? the error isn't likely to be actionable for a node operator if our backend goes offline